test/common/common/safe_memcpy_test.cc

```diff
@@ -0,0 +1,39 @@
+#include <array>
+
+#include "common/common/safe_memcpy.h"
+#include "common/grpc/common.h"
+#include "common/http/headers.h"
+#include "common/http/message_impl.h"
+#include "common/http/utility.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+using namespace testing;
```

Comment:
we typically don't import the entire namepace. Instead we just import the symbols we need. In your case I think it's just
```
using testing::Eq;
```